Add Synaltic to users page

yabir opened the pull request.

gunnarmorling commented:
Very nice, thanks a lot! Could you only rework the commit so to keep the list in alphabetic order and also author the commit so it uses an e-mail associated to your GitHub account? Thanks again!

yabir commented:
Thanks for the quick reply @gunnarmorling ! I dropped the first commits and Added another one with the modifications you suggested + Synaltic's blog post

gunnarmorling commented:
Applied, thx a lot. Will be live in a few min.
